packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -463,16 +464,14 @@ const DateRangeCalendar = React.forwardRef(function DateRangeCalendar<TDate>(
     [calendars],
   );
 
-  const visibleMonths = React.useMemo(
-    () =>
-      Array.from({ length: calendars }).map((_, index) =>
-        utils.setMonth(
-          calendarState.currentMonth,
-          utils.getMonth(calendarState.currentMonth) + index,
-        ),
-      ),
-    [utils, calendarState.currentMonth, calendars],
-  );
+  const visibleMonths = React.useMemo(() => {
+    const firstMonth =
+      calendarDirection === 'onward'
```

Comment:
This option specifically doesn't cover the condition mentioned in the issue of rendering the current month in the middle of 3 months...
What do you think about going with a simple number denoting the location of the current month in the calendars list? 🤔 
For example:
```tsx
calendars={3}
currentMonthCalendarPosition={2}
```
would produce a result of 3 calendars, where the second (middle) one would be the current month.
I'd think that the prop should be 1-indexed in this case, in order to be easier to reason about, given the `calendars` prop behavor.
Of course, the prop name is just a suggestion. 🙈 🤷

Reply:
I tried to go with something higher level than the index because I think it's slightly easier to understand.
But that's probably too complicated to cover all the use cases with it indeed...

The challenge if we use the index is to have a meaningful prop name.
Because the "current month" is an internal notion that probably don't make a lot of sense for the user on a range picker, they just see 2 "current" months (or 1 or 3 if they want to) .

Reply:
Indeed, a clear enough prop name is the biggest problem in this case. 🤔 
Although, I think that my suggested prop name is one of the more clear ones, especially given a clear enough prop description. 🙈 🤷 
At the moment I can't come up with any better suggestion... Feel free to suggest anything if yo have options. 🙏

Reply:
I don't have a better idea tbh